examples/src/main/r/RSparkSQLExample.R

```diff
@@ -18,31 +18,43 @@
 library(SparkR)
 
 # $example on:init_session$
-sparkR.session(appName = "MyApp", sparkConfig = list(spark.executor.memory = "1g"))
+sparkR.session(appName = "MyApp", sparkConfig = list(spark.some.config.option = "some-value"))
```

Comment:
It's just an example for how to set extra configuration options. It's not read anywhere.